content/en/docs/tasks/access-application-cluster/configure-dns-cluster.md

```diff
@@ -5,7 +5,7 @@ content_type: concept
 ---
 
 <!-- overview -->
-Kubernetes offers a DNS cluster addon, which most of the supported environments enable by default. In Kubernetes version 1.11 and later, CoreDNS is recommended and is installed by default with kubeadm.
+Kubernetes offers a DNS cluster addon, which most of the supported environments enable by default. In Kubernetes version 1.11 and later, CoreDNS is recommended and is installed by default with kubeadm, minikube and kind clusters as well.
```

Comment:
> As much as i know coredns is installed by default since minikube 0.29
Yes you're right

> Is v1.11 the default k8s version for this particular minikube version ?
Well minikube v0.29 was released back in September 2018, with default stable k8s version set to v1.12.0
While kind was first released November 2018, with default stable k8s version set to v1.12.3

kind from the very beginning has default support of CoreDNS I think.
[issue#30](https://github.com/kubernetes-sigs/kind/issues/30)